pkg/doc/jose/jws.go

```diff
@@ -0,0 +1,380 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package jose
+
+import (
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/square/go-jose/v3"
+	"github.com/square/go-jose/v3/json"
+)
+
+const (
+	jwsPartsCount    = 3
+	jwsHeaderPart    = 0
+	jwsPayloadPart   = 1
+	jwsSignaturePart = 2
+)
+
+// Headers represents JOSE headers.
+type Headers map[string]interface{}
+
+// JWK (JSON Web Key) is a JSON data structure that represents a cryptographic key.
+type JWK jose.JSONWebKey
+
+// GetJWK gets JWK from JOSE headers.
+func (jh Headers) GetJWK() (*JWK, bool) {
```

Comment:
Why do we prefix these methods with Get? We don't seem to be getting anything.

Reply:
Agree. I will rename it to `JWK()`. The same applied for the `KeyID()` and `Algorithm()`.